config/routes.rb

```diff
@@ -5,6 +5,7 @@
 
   resource :dashboard, only: [:index]
   resource :updates, only: [:create]
+  resources :nodes, only: [:destroy]
```

Comment:
with having a `NodesController` do you think we can also add the bootstrap api to its responsibility? so `destroy` is a member method and `bootstrap` is a collection method